content/partials/shared/_channel_namespaces.textile

```diff
@@ -1,11 +1,17 @@
-One or more channel namespaces, or channel name prefixes, may be "configured for an app in your dashboard":https://support.ably.com/solution/articles/3000030053-how-do-i-access-my-app-dashboard. When a channel is created whose name is prefixed with one of the namespaces, the channel assumes certain configured attributes associated with that namespace. For example, a channel namespace named "@private@" would match channels named "@private@", "@private:chat@", "@private:chat:mike@".
```

Comment:
Could the Freshdesk article be replaced with the following https://knowledge.ably.com/how-do-i-access-my-app-dashboard-1 ?